### DISCUSSION TITLE
Why might I see (nearly) identical unusual solutions?

### DISCUSSION THREAD

**Discussion by ChairmanMawh** [codewars/codewars.com#2935] [General]
Browsing the solutions to [this kata](https://www.codewars.com/kata/5d5ee4c35162d9001af7d699) I saw two near identical solves, posted close to each other (solutions sorted by time), even to the exact figure of arbitrarily chosen starting min constants, variable names etc.. What reward is being reaped by behaving thus?

![image](https://user-images.githubusercontent.com/118385137/217520117-8f47da55-f7c0-408e-8ea8-3ec2f6813e5e.png)

**Comment by hobovsky** [comment]
Well, now after posting the screenshots, you can expect a hundred of nearly identical solutions submitted very soon in a rapid succession :)

That's where they come from.
